pkg/printers/internalversion/printers.go

```diff
@@ -60,7 +59,20 @@ import (
 	"k8s.io/kubernetes/pkg/util/node"
 )
 
-const loadBalancerWidth = 16
+const (
+	loadBalancerWidth = 16
+
+	// LabelNodeRoleMaster specifies that a node is a master
+	// It's copied over to here until it's merged in core: https://github.com/kubernetes/kubernetes/pull/39112
```

Comment:
Yes, pkgs in `cmd` can never be referenced from `pkg`.  @deads2k we should make sure there is an import rule for this.